scripts/macos-sign-binaries.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# intended to run on MacOS only
+# signs all executable files in a given folder (as $1) with developer certificate
+
+# required variables:
+# APPLE_DEVELOPER_IDENTITY: "Developer ID Application: <company name> (<hex id>)"
+# APPLE_DEVELOPER_ID_BUNDLE: base64-encoded content of apple developer id certificate bundle in pksc12 format
+# APPLE_DEVELOPER_ID_BUNDLE_PASSWORD: password used when exporting the bundle
```

Comment:
if this is to be a part of the CI,
should we add them to github secrents ? or should we run int in a private self-hosted runner ?

Reply:
github secrets should work